src/plugins/index_pattern_management/public/components/create_index_pattern_wizard/components/step_index_pattern/step_index_pattern.tsx

```diff
@@ -39,6 +39,7 @@ import {
 } from '@elastic/eui';
 import { i18n } from '@osd/i18n';
 import { FormattedMessage } from '@osd/i18n/react';
+import _ from 'lodash';
```

Comment:
Nit: you can import just the debounce function here.

```suggestion
import { debounce } from 'lodash';
```

Reply:
thanks, I'll fix it. :)